nav2_gps_waypoint_follower_demo/launch/mapviz.launch.py

```diff
@@ -0,0 +1,33 @@
+import launch
+import launch.actions
+import launch.substitutions
+import launch_ros.actions
+import os
+from ament_index_python.packages import get_package_share_directory
+
+gps_wpf_dir =  get_package_share_directory("nav2_gps_waypoint_follower_demo")
+mapviz_config_file = os.path.join(gps_wpf_dir, "config", "gps_wpf_demo.mvc")
+
+def generate_launch_description():
```

Comment:
Why not have this launched in `gps_waypoint_follower.launch.py`?

Reply:
I think it can be conditionally launched with a launch argument. would that be ok? This is because there are some sections in the tutorial that do not require mapvix

Reply:
Yeah conditional is fine!